plugin/mouse/mouse_scroll.py

```diff
@@ -7,6 +8,7 @@
 scroll_job = None
 gaze_job = None
 scroll_dir: Literal[-1, 1] = 1
+scroll_ts: float = 0
```

Comment:
Thought about suggesting `scroll_start_ts` which is a tiny bit clearer but uglier. Not strongly opinionated. Feel free to go either way.

Reply:
done